src/KafkaFlow/Extensions/MessageContextExtensions.cs

```diff
@@ -0,0 +1,10 @@
+namespace KafkaFlow.Extensions;
+
+internal static class MessageContextExtensions
+{
+    public static void Discard(this IConsumerContext context)

```

Comment:
Why an extension method? and not on the ConsumerContext itself?

Reply:
I believe this method meets the requirements of an extension method, as it solely utilizes the public members of the `IConsumerContext`. Therefore, any implementation of this interface will have the capability to use the method.